test/fixtures/change-code.js

```diff
@@ -77,7 +79,7 @@ function listener (code, signal) {
 function run (opt) {
   console.error(opt)
   var shell = process.platform === 'win32' ? null : { shell: '/bin/bash' }
-  exec(process.execPath + ' ' + __filename + ' ' + opt, shell, function (err, stdout, stderr) {
+  exec(join(process.execPath, ' ', __filename, ' ' + opt), shell, function (err, stdout, stderr) {
```

Comment:
100% agree this is a weird new standard nit that I was working around -- if it sees something that looks like a path being execed, it yells at you until you use path.resolve or path.join.